packages/core/src/App/Components/Layout/Header/account-info-icon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+type TAccountInfoIcon = {
+    is_virtual?: boolean;
+    currency?: string;
+};
+
+const AccountInfoIcon = ({ is_virtual, currency }: TAccountInfoIcon) => (
+    <Icon
+        data_testid='dt_icon'
+        icon={`IcCurrency-${is_virtual ? 'virtual' : currency || 'Unknown'}`}
```

Comment:
Hi @sergei-deriv, shouldn't this be `IcCurrency${is_virtual ? 'virtual' : currency || 'Unknown'}`? Please remove the `-` 🙏.

Reply:
@aum-deriv I'm pretty sure you are right, but it's legacy code and it used in production for DTraderHeader, not just for wallet project. It's really strange how but it works, this is legacy code which I just moved to a separate component uses in `AccountInfo` and in `AccountInfoWallets`. I'll recheck it with usual account (not wallets) and I'll make the decision

Reply:
@aum-deriv It doesn't work when I delete this dash

<img width="989" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/175d2ee4-7821-47dd-bfdd-39bef4e7b8df">

So I need to leave it